cra-tf-validate-ignore-rules.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scc_rules": [
+    {
+      "scc_rule_id": "rule-8cbd597c-7471-42bd-9c88-36b2696456e9",
+      "description": "Check whether Cloud Object Storage network access is restricted to a specific IP range",
+      "ignore_reason": "This is restricting by CBR, not IP range. This is not yet checked by the scan Tracked: AHA SCC-961",
```

Comment:
Perhaps lets be a little clearer here...

This module supports restricting network access using Context Based Restrictions (CBRs), however SCC does not yet support scanning for CBR rules, so hence the rule currently fails. SCC CBR support is being tracked in AHA SCC-961